src/native_mux_impl/stdio.rs

```diff
@@ -1,12 +1,13 @@
-use crate::stdio::StdioImpl;
-use crate::Error;
-use crate::Stdio;
+use crate::{stdio::StdioImpl, Error, Stdio};
 
-use once_cell::sync::OnceCell;
+use std::{
+    fs::{File, OpenOptions},
+    io,
+    os::unix::io::{AsRawFd, FromRawFd, IntoRawFd, OwnedFd, RawFd},
```

Comment:
This bumps MSRV by a decent amount, which is a little unfortunate, but not the end of the world.

Reply:
We are already using `OwnedFd` in `impl From<OwnedFd> for Stdio`, so IMO it's fine.